### ISSUE TITLE
"pkg-config --variable=builtin_h265_decoder libheif" does not return yes

### ISSUE THREAD

**Issue by hornschorsch** [strukturag/libheif#1299]
Tody i tried to compile gimp with support for heic/heif files on my slackware 15.0 system. I installed libde265 and x265 and then built libheif and it correctly statet:

```
=== Supported formats ===
format        decoding   encoding
HEIC            YES        YES
```

The configure script of gimp then does the following checks:

```
can_import_heic=`$PKG_CONFIG --variable=builtin_h265_decoder libheif`
can_export_heic=`$PKG_CONFIG --variable=builtin_h265_encoder libheif`
```

but "pkg-config --variable=builtin_h265_encoder libheif" does not return "yes", despite the fact that heic encoding and decoding is enabled.

I replaced the two lines with

```
can_import_heic=yes
can_export_heic=yes
```

and then i could compile gimp with heic support and it really can load and save heic images then.

What am i missing?

**Comment by hornschorsch** [comment]
Please see my comment at https://github.com/strukturag/libheif/issues/758#issuecomment-2329501979

**Comment by farindk** [comment]
We're not going to reintroduce these pkg-config variables because they make no sense for the way current versions of libheif work.

**Comment by fancycode** [comment]
See https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1078402#24 for a possible fix of the detection code in Gimp 2.10.
